lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,22 +1,59 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const Tab = props => {
+
+  
+class Tab extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      tab: '',
+    }
+  }
+
+  componentDidMount(){
+    this.setState({tab: this.props.selectedProp})
+    console.log(this.props.selectedProp)
+  }
+
+  
+
+  render(){
+    console.log(this.props.selectTabHandler)
+    if (this.props.selectedProp === this.props.selectedTab) {
+      return (
```

Comment:
Instead or return two different `divs` here you could just do one because they are pretty much the same thing and then do a ternary statement to conditionally select the `className`